File: task-03/sub-task 3/Python/code.py
n = int(input())
for i in range((n + 1) // 2):
    print(" " * ((n // 2) - i) + "*" * (2 * i + 1))
for i in range((n // 2) - 1, -1, -1):
    print(" " * ((n // 2) - i) + "*" * (2 * i + 1))

File: task-03/sub-task 3/Javascript/code.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Enter an odd number of rows for the diamond: ", n => {
    n = parseInt(n);
    
    for (let i = 0; i <= Math.floor(n / 2); i++) {
        console.log(" ".repeat(Math.floor(n / 2) - i) + "*".repeat(2 * i + 1));
    }

    for (let i = Math.floor(n / 2) - 1; i >= 0; i--) {
        console.log(" ".repeat(Math.floor(n / 2) - i) + "*".repeat(2 * i + 1));
    }
    
    readline.close();
});
